addons/account_analytic_default/models/account_move.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
+class AccountMove(models.Model):
+    _inherit = 'account.move'
```

Comment:
```suggestion
    _inherit = 'account.move.line'
```
______

If we add a constraint here, I fear that there will be errors when transforming a Sales/Purchase Order into an invoice might become impossible.

______

It might also become annoying for people who already have tags setup that way, they won't be able to change anything unless they fix it. So maybe not the best idea in stable.

Reply:
how does applying the constraint to `account.move.line` change that ?

Reply:
These are 3 separate points

Reply:
> not the clearest comment sorry.

remove this